src/main/java/io/vertx/core/file/impl/FileResolverImpl.java

```diff
@@ -237,6 +237,7 @@ private File unpackUrlResource(URL url, String fileName, ClassLoader cl, boolean
       case "bundleresource": // Equinox
       case "jrt": // java run-time (JEP 220)
       case "resource":  // substratevm (graal native image)
+      case "vfs":  // jboss-vfs
```

Comment:
this seems very specific and not something we need to support

Reply:
well we can support this as best effort to some extend, i.e without extra specific code

Reply:
I'm just worried if we do support this, then we'll have to add specific custom code to handle this

Reply:
Agree to support this as best effort to an extend without extra code changes :)

If there are other vfs related issues after this, it should be done by a customized FileResolver.